web/src/components/grouped-score-badge.tsx

```diff
@@ -5,88 +5,161 @@ import {
   HoverCardTrigger,
 } from "@/src/components/ui/hover-card";
 import {
-  isBooleanDataType,
-  isCategoricalDataType,
-} from "@/src/features/manual-scoring/lib/helpers";
-import { type ScoreSimplified } from "@/src/server/api/routers/generations/getAllQuery";
+  type QuantitativeAggregate,
+  type QualitativeAggregate,
+} from "@/src/features/manual-scoring/lib/aggregateScores";
+import { type APIScore } from "@/src/features/public-api/types/scores";
+
+import { numberFormatter } from "@/src/utils/numbers";
 import { cn } from "@/src/utils/tailwind";
+import { type Score } from "@langfuse/shared";
 import { MessageCircleMore } from "lucide-react";
 
-export const GroupedScoreBadges = ({
-  scores,
-  variant = "badge",
+const COLOR_MAP = new Map([
+  ["True", "bg-light-green p-0.5 text-dark-green"],
+  ["False", "bg-light-red p-0.5 text-dark-red"],
+]);
+
+const SingleScoreValue = ({
+  value,
+  comment,
+  showColorCoding = false,
 }: {
-  scores: ScoreSimplified[];
-  variant?: "badge" | "headings";
+  value: string;
```

Comment:
why is this string and not type aware {scoreDataType, scoreValue: string|number}

Reply:
drop as we do not use this independently